### PR TITLE
refactor: remove easyjson

### PR DESCRIPTION
**Pull request by fabriziosestito**
## Description

Removes easyjson, bumps  dependencies, and GHA.

## Test

Existing tests should pass.
An e2e test was modified, see comments.

### CODE REVIEW

File: e2e.bats [hunk 1]
```diff
@@ -28,7 +28,7 @@
   # check if patch operation
   patch=$(echo "${output}" | grep '{.*\"patch\".*}'| jq -r ".patch" | base64 --decode)
   echo "patch=$patch"
-  echo ${patch} | jq -e "[.[].op == \"add\" and .[].value.cccenter == \"zpto\"] | any"
+  echo ${patch} | jq -e "[.[].op == \"add\" and .[].value.cccenter? == \"zpto\"] | any"
```
[the remark this answers is not in the document]
Comment (jvanz) [reply e2e.bats:31]: I'm wondering, why is the patch adding metadata fields? Should not it just add the label?

**Review by jvanz — COMMENTED** [review]
The change looks good. But I just have one question.

File: e2e.bats (hunk 1 above)
[the remark this answers is not in the document]
Comment (viccuad) [reply e2e.bats:31]: I think this was a bug. Nevertheless, we only show the patch on `kwctl run`, as a help for policy developers.
I suppose easyjson did not provide timestamps, which are irrelevant. I'm ok leaving or dropping that metadata path altogether.

**Review by viccuad — APPROVED** [review]
LGTM! I'm ok leaving the patch output as it is, unless there's a simple way to tell it to skip the metadata path.

File: e2e.bats (hunk 1 above)
[the remark this answers is not in the document]
Comment (flavio) [reply e2e.bats:31]: I think it's fine to leave the patch output in this way

**Review by flavio — APPROVED** [review]
LGTM, I think we can merge this PR

**Pull request merged**